src/server/api/routers/generations.ts

```diff
@@ -88,92 +88,99 @@ export const generationsRouter = createTRPCRouter({
         >
       >(
         Prisma.sql`
-          WITH observations_with_latency AS (
-            SELECT
-              o.*,
-              CASE WHEN o.end_time IS NULL THEN NULL ELSE (EXTRACT(EPOCH FROM o."end_time") - EXTRACT(EPOCH FROM o."start_time"))::double precision END AS "latency"
-            FROM observations o
-            WHERE o.type = 'GENERATION'
-            AND o.project_id = ${input.projectId}
-            ${datetimeFilter}
-          )
+        WITH observations_with_latency AS (
```

Comment:
When testing the feature, i get the following error:

`TRPCClientError: Invalid `prisma.$queryRaw()` invocation: Raw query failed. Code: `22P03`. Message: `ERROR: incorrect binary data format in bind parameter 6`

This happens, when i try to filter by decimal values.

The input vallues to the API look good:


```
{
    "page": 0,
    "limit": 50,
    "projectId": "7a88fb47-b4e2-43b8-a06c-a5ce950dc53a",
    "filter": [
        {
            "column": "timestamp",
            "operator": ">",
            "value": "2024-01-08T00:00:00.000Z",
            "type": "datetime"
        },
        {
            "type": "numberObject",
            "column": "scores_avg",
            "key": "feedback",
            "operator": "<",
            "value": 0.01
        }
    ],
    "searchQuery": null,
    "orderBy": {
        "column": "timestamp",
        "order": "DESC"
    }
}
```
Somewhere in the SQL code, we fail with the double value.